source/code/plugins/out_oms_changetracking_file.rb

```diff
@@ -0,0 +1,591 @@
+module Fluent
+
+  class OutChangeTrackingFile < BufferedOutput
+
+    Plugin.register_output('out_oms_oms.changetracking.file', self)
```

Comment:
there is no plugin naming convention, as i understand. we have plugins with all sorts of name for pluigns.
you have ones with _ and .

the reason this is "out_oms_oms.changetracking.file", becuase it is a generated from Tag "oms.changetracking.file"
to change this will mean changing the tag for entire change tracking system (service, daemons as well)